packages/account/src/Sections/Profile/PersonalDetails/personal-details.jsx

```diff
@@ -1084,10 +1084,7 @@ export const PersonalDetailsForm = ({
                                     <>
                                         <div className='account-form__divider' />
                                         <div className='pro-client'>
-                                            <FormSubHeader
-                                                className='account-form__red-header'
```

Comment:
@yauheni-kryzhyk-deriv did you intend to remove this className? could you please explain the reason? 🙂